hieradata/site/cp/role/htcondor.yaml

```diff
@@ -0,0 +1,14 @@
+---
+htcondor::htcondor_version: "23.0"
```

Comment:
It looks like the version can be factored out to the top level class too.

Reply:
your right, versions should stay the same and it needs a different on dev, it can be overwritten at site